alvr/server_core/src/tracking/mod.rs

```diff
@@ -124,7 +121,8 @@ impl TrackingManager {
     // Performs all kinds of tracking transformations, driven by settings.
     pub fn report_device_motions(
         &mut self,
-        config: &HeadsetConfig,
+        headset_config: &HeadsetConfig,
+        connection_config: &ConnectionConfig,
```

Comment:
All the other usage instances of statistics_history_size are not changing in real time. Mainly because they are tied to the initialization of sliding average objects. So you don't need to pass this parameter every time. Check how it's handled in StatisticsManager

Reply:
Resolved